### PR TITLE
Create the complete single page math app

### PR DESCRIPTION
**Pull request by lawrahkonwea**
This is the complete math magician app, it was created using CRA;
- I added style to improve the look and feel using React basic styling.
- Every page had a route using React router, linking all pages to each other.
- The navigation links for all three  pages were displayed on each page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,24 @@
 import React, { Component } from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Header from './components/navbar';
+import HomePage from './components/home';
+import Quote from './components/quote';
 import CalcUserInterface from './components/calculator';
 
 // eslint-disable-next-line
```
Comment (zieeco) [inline src/App.js]: - Please, let's kindly remove commented codes from our codebase to make it clean and maintainable.

Comment (zieeco) [reply src/App.js]: #### README FILE


- Please, let's kindly remove all these placeholder texts as this text was added to help you add the right content in the right section of the readme when updating the content in the template to suit your project needs. Please, kindly remove them to make your readme more professional. See them  below:


Screenshot from 2023-01-26 16-24-26
Screenshot from 2023-01-26 16-24-19
Screenshot from 2023-01-26 16-24-09
Screenshot from 2023-01-26 16-23-59
Screenshot from 2023-01-26 16-23-49

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @lawrahkonwea ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @lawrahkonwea , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
All linters are passing :heavy_check_mark: 
Great job so far :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**